docs/latest/tutorial/tutorial-node-access.md

```diff
@@ -0,0 +1,260 @@
+    webPreferences: {
+      preload: path.join(__dirname, 'preload.js'),
+    },
+  });
+
+  win.loadFile('index.html');
+}
+
+app.whenReady().then(() => {
+  createWindow();
+});
+```
+
+:::tip
+There are two Node.js concepts that are used here:
+
+- The [`__dirname`][dirname] string points to the path of the currently executing script
+  (in this case, your project's root folder).
+- The [`path.join`][path-join] API joins multiple path segments together, creating a
+  combined path string that works across all platforms.
+
+If you are not familiar with Node.js, we recommend you to read [this guide][node-guide].
+:::
+
+At this point, the renderer context has access to `versions`, so let's make that information
+available to the user. Create a new file `renderer.js` that contains the following code:
+
+```js
+const information = document.getElementById('information');
+information.innerText = `This app is using Chrome (v${versions.chrome()}), Node.js (v${versions.node()}), and Electron (v${versions.electron()})`;
+```
+
+And modify `index.html` to match the following:
+
+```html {18,20}
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      http-equiv="Content-Security-Policy"
+      content="default-src 'self'; script-src 'self'"
+    />
+    <meta
+      http-equiv="X-Content-Security-Policy"
+      content="default-src 'self'; script-src 'self'"
+    />
+    <title>Hello from Electron renderer!</title>
+  </head>
+  <body>
+    <h1>Hello from Electron renderer!</h1>
+    <p>👋</p>
+    <p id="info"></p>
+  </body>
+  <script src="./renderer.js"></script>
+</html>
+```
+
+We have created a paragraph identified with `information` and included `renderer.js`.
+
+:::warning
+The reason why we do not write the JavaScript directly in the HTML between `<script></script>`
+is because we have a Content-Security-Policy that will prevent its execution.
+To know more you can visit [MDN's CSP content][mdn-csp].
+:::
+
+After following the above steps, you should have a fully functional Electron application that
+has a message similar to the following one (probably with different versions):
+
+> This app is using Chrome (v94.0.4606.81), Node.js (v16.5.0), and Electron (v15.2.0)
+
+And the code should be similar to this:
+
+```fiddle docs/latest/fiddles/quick-start
+
+```
+
+## Debugging the renderer process
+
+At this moment you have code being executed in the main and renderer process.
+The easiest way to debug the code in your renderer process is by using the built-in developer
+tools. You can invoke them with a shortcut (`Ctrl/Command+Shift+I`) or programmatically
+via `mainWindow.webContents.openDevTools()` in the main process.
+
+If you want to use VS Code to debug the main and renderer process simulatenously from the
+same tool, you have to use a [compound task].
+To do so, open the file `.vscode/launch.json` that you created in the previous step and update
+it to look as follows (changes are highlighted):
```

Comment:
Don't know if this fits into the `Accessing Node.js APIs` section of the tutorial

Reply:
What do you think of adding some JS in [Adding some UI](https://electronjsorg-new-pr-124.herokuapp.com/docs/latest/tutorial/scaffolding#adding-some-ui) and add this debugging part there as well? That way all the VS Code debugging is in the same section.

Reply:
Moved all the debugging and VS Code stuff to the scaffolding step